src/Server/passport/lib/app/lifecycle.js

```diff
@@ -156,8 +156,9 @@ module.exports.afterSocialAuth = provider =>
         err.response.body &&
         err.response.body.error === "Another Account Already Linked"
       ) {
-        alreadyLinkedParams = `?error=already-linked&provider=${providerName}`
-        return res.redirect(opts.settingsPagePath + alreadyLinkedParams)
+        return res.redirect(
+          `${opts.settingsPagePath}?error=already-linked&provider=${providerName}`
```

Comment:
I don't think this redirect is going to actually _do_ anything... but it's better than 500ing...

Reply:
OK I'll just fix this too.